pkg/api/testing/fuzzer.go

```diff
@@ -365,6 +365,10 @@ func FuzzerFor(t *testing.T, version string, src rand.Source) *fuzz.Fuzzer {
 			s.MinReplicas = &minReplicas
 			s.CPUUtilization = &extensions.CPUTargetUtilization{TargetPercentage: int(c.RandUint64())}
 		},
+		func(g *unversioned.GroupVersion, c fuzz.Continue) {
+			g.Version = strconv.FormatUint(c.RandUint64(), 10)
```

Comment:
Please put strings and not numbers into these fields. (do you really need this fuzz function?)